models/ga4_export__conversions_report.sql

```diff
@@ -25,7 +16,7 @@ with events_base as (
 
     select * 
     from {{ ref('stg_ga4_export__event') }}
-    where event_name in ({{ "'" ~ var('key_events') | join("', '") ~ "'" }})
+    where event_name in ({{ "'" ~ var('key_events',[]) | join("', '") ~ "'" }})
```

Comment:
I know you set the default in the dbt_project.yml, but it's best to also include default in line.